jmx-scraper/src/integrationTest/java/io/opentelemetry/contrib/jmxscraper/JmxConnectorBuilderTest.java

```diff
@@ -31,9 +32,8 @@ static void afterAll() {
 
   @Test
   void noAuth() {
-    int port = PortSelector.getAvailableRandomPort();
```

Comment:
we can reuse armeria implementation as it's already included in classpath here.

Reply:
Hmm, I'd rather avoid using this class - this is internal class and is not even mentioned in their official API docs: https://javadoc.io/doc/com.linecorp.armeria/armeria-javadoc/latest/index.html

Reply:
Let's keep the class then, we could also reuse the one in instrumentation later if there are other usages to prevent duplication at the cost of adding another test dependency.